README.md

```diff
@@ -29,11 +33,15 @@ If you do, please install NSS (example for Debian / Ubuntu):
 sudo apt-get install libnss3
 ```
 
+</details>
+
 ## Development
 
 If you want to hack on this project, here is how you do it.
 
-#### Building the sources
+<details><summary>Expand</summary>
```

Comment:
The "Expand" thing looks a little confusing to me. Any ideas for more meaningful summary? Maybe something like "Build the project"?

Reply:
I merged it because if I push another commit I'd need to wait for approval again :-P We'll update it before the release.